openpype/tools/publisher/publish_report_viewer/window.py

```diff
@@ -148,9 +201,66 @@ def update_file_content(self):
         self.content = content
         self.file_modified = file_modified
 
+    @staticmethod
+    def date_obj_to_timestamp(date_obj):
+        if hasattr(date_obj, "timestamp"):
+            return date_obj.timestamp()
+
+        # Python 2 support
+        epoch = datetime.datetime.fromtimestamp(0)
+        return (date_obj - epoch).total_seconds()
+
+    @classmethod
+    def _fix_content(cls, content, file_modified=None):
+        """Fix content for backward compatibility of older report items.
+
+        Args:
+            content (dict[str, Any]): Report content.
+            file_modified (Optional[float]): File modification time.
+
+        Returns:
+            bool: True if content was changed, False otherwise.
+        """
+
+        # Fix created_at key
+        changed = cls._fix_created_at(content, file_modified)
+
+        # NOTE backward compatibility for 'id' and 'report_version' is from
+        #    28.10.2022 https://github.com/ynput/OpenPype/pull/4040
+        # We can probably safely remove it
+
+        # Fix missing 'id'
+        item_id = content.get("id")
+        if not item_id:
+            item_id = str(uuid.uuid4())
+            changed = True
+            content["id"] = item_id
+
+        # Fix missing 'report_version'
+        if not content.get("report_version"):
+            changed = True
+            content["report_version"] = "0.0.1"
+        return changed
+
+    @classmethod
+    def _fix_created_at(cls, content, file_modified):
+        # Key 'create_at' was added in report version 1.0.1
+        created_at = content.get("created_at")
+        if created_at:
+            return False
+
+        # Auto fix 'created_at', use file modification time if it is not set
+        #   , or current time if modification could not be received.
```

Comment:
Odd sentence start with the `,` on a blank line. Can we just remove the `,`?

Reply:
```suggestion
        # Auto fix 'created_at', use file modification time if it is not set
        #   or current time if modification could not be received.
```